templates/_asset_fields.phtml

```diff
@@ -66,12 +66,14 @@ $_asset_values = array_merge([
                 'GPLv3' => 'GPL v3',
                 'GPLv2' => 'GPL v2',
                 'AGPLv3' => 'AGPL v3',
+                'Apache 2.0' => 'Apache 2.0',
```

Comment:
I think the lvalue is meant not to have any space and/or to follow SPDX, so better use `Apache-2.0`.